tests/auth_tests/test_forms.py

```diff
@@ -139,7 +139,9 @@ class Meta(UserCreationForm.Meta):
         form = CustomUserCreationForm(data)
         self.assertTrue(form.is_valid())
 
-
+@override_settings(
```

Comment:
It would be helpful to add an explanatory comment like "To verify that the login form rejects inactive users, use an authentication backend that allows them." Something similar for the other places you modified might be useful as well.